client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -33,8 +35,11 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-function CreateMeetup() {
+function CreateMeetup({ id }) {
+  // if id is null, create new meetup, else fetch data for meetup/:id and edit that meetup
   const classes = useStyles();
+  const user = useSelector((state) => state);
```

Comment:
We don't need the entire user.

```suggestion
  const userID = useSelector((state) => state.id);
```